Formula/cryptominisat.rb

```diff
@@ -12,11 +11,16 @@ class Cryptominisat < Formula
   end
 
   depends_on "cmake" => :build
+  # fix `'stdio.h' file not found` on macosx-10.13
+  depends_on :xcode => ["11.0", :build]
```

Comment:
Does it actually require XCode? Or just sierra or higher?

Reply:
yeah, I can do mojave, now I think that might be a better way to solve this issue.